.github/workflows/tests.yml

```diff
@@ -36,18 +46,21 @@ jobs:
         with:
           auto-update-conda: true
           miniforge-variant: Mambaforge
-          channels: conda-forge
-          environment-file: environment-dev.yml
+          environment-file: ${{ matrix.env }}
           use-mamba: true
 
       - name: Install pyjanitor
         run: python -m pip install -e .
 
       - name: Run docstrings tests
+        if: ${{ contains(matrix.env, 'latest.yaml') && contains(matrix.os, 'ubuntu') }}
         run: pytest -v -r a -n auto --color=yes --durations=0 --cov=janitor --cov-append --cov-report term-missing --cov-report xml --doctest-only janitor
 
-      - name: Run unit tests
-        run: pytest -v -r a -n auto --color=yes --durations=0 --cov=janitor --cov-append --cov-report term-missing --cov-report xml tests -m "${{ matrix.test-subset }}"
+      - name: Run not turtle unit tests
+        run: pytest -v -r a -n auto --color=yes --durations=0 --cov=janitor --cov-append --cov-report term-missing --cov-report xml tests -m "not turtle"
+
+      - name: Run turtle unit tests
+        run: pytest -v -r a -n auto --color=yes --durations=0 --cov=janitor --cov-append --cov-report term-missing --cov-report xml tests -m "turtle"
```

Comment:
Historically we've been able to reduce testing time by running the `turtle` and `not turtle` tests in parallel. Codecov will also get updated correctly as long as `--cov-append` is called, if I remember correctly. Can we reinstate running `turtle` and `not turtle` tests here, please?